src/auth/index.ts

```diff
@@ -44,14 +48,13 @@ export async function login(
   }
 }
 
-export async function changePassword(
-  username: string,
+async function changePassword(
+  usr: User,
   pwchangeinput: ChangeUserPasswordInput
```

Comment:
Could do some object destructing to clean up the code a bit?

```
{currentPassword, newPassword}: ChangeUserPasswordInput
```

This will allow you to just refer to the properties you need in the function.